src/DataModel/IClass.ts

```diff
@@ -0,0 +1,5 @@
+import {IResource} from "./IResource";
+
+export interface IClass extends IResource
```

Comment:
Again, I don't think the code needs (or even should) match the representations on the wire 1:1